install/browser/browser-docs/install.yml

```
@@ -0,0 +1,14 @@
+id: browser-docs
+name: Browser
+title: Browser Monitoring
+description: |
+  With New Relic One's browser monitoring solution, you get full visibility into the complete webpage life cycle of your application or website. Browser measures page load timing, also known as Real User Monitoring (RUM).
+
+target:
+  type: integration
+  destination: cloud
```

Is browser technically an agent? I'd think to make this:

```suggestion
  type: agent
  destination: application
```

Maybe that's wrong though ¯\_(ツ)_/¯